Source/WebCore/platform/graphics/cocoa/SourceBufferParserWebM.cpp

```diff
@@ -489,7 +489,7 @@ class MediaDescriptionWebM final : public MediaDescription {
     const webm::TrackEntry& track() { return m_track; }
 
 private:
-    mutable std::optional<AtomString> m_codec;
+    mutable std::optional<String> m_codec;
```

Comment:
values are lazily constructed because it's using AtomString. So internally, those objets either use a std::string or a NSString ; it is assume that the thread on which the object is read, is the only one that will ever be read.

But that assumption will not hold for long.

What I will do is always generate the string on the fly then, so you get a new string.

Using a non-thread safe refcount isn't a workable solution